extensions/oidc-client-filter/runtime/src/main/java/io/quarkus/oidc/client/filter/runtime/OidcClientFilterConfig.java

```diff
@@ -11,12 +11,16 @@ public class OidcClientFilterConfig {
     /**
      * Enable OidcClientRequestFilter for all the injected MP RestClient implementations.
      * If this property is disabled then OidcClientRequestFilter has to be registered as an MP RestClient provider.
+     * You can use this property in combination with the `@OidcClientFilter` annotation, as sometimes, you might need to
```

Comment:
IMHO it is not worth highlighting such an option - register the default or configured client filter for all the rest clients except those which have named client filters - as it can be confusing. Perhaps the simplest option is to change. I'm not sure really how to say here without confusing the users, sorry I even asked to update this specific doc, I guess the existing line `If this property is disabled then OidcClientRequestFilter has to be registered as an MP RestClient provider.` is descriptive enough since `@OidcClientFilter` is about registering it as a provider.
Please just remove this added line - sorry about it

Reply:
Np, for me it's tough thing to write docs.